Automattic/jetpack
# Dashboard: Remove the user license activation notice

#### Changes proposed in this Pull Request:

* This PR removes the user license activation notice that displays on the Jetpack dashboard if a user has one or more unattached licenses.
* The notice is being replaced by a JITM once Jetpack 11.6 is active on a site - more information here: p9dueE-65G-p2 

#### Other information:


#### Jetpack product discussion

p9dueE-65G-p2

#### Does this pull request change what data or activity we track or use?

* The tracks events `jetpack_wpa_licensing_activation_notice_view`, `jetpack_wpa_licensing_activation_notice_dismiss`, and tracks clicks `licensing_activation_notice` and `calypso_purchases_link` will be removed, but are replaced with similar events in the JITM. So in that respect, no privacy doc updates needed.

#### Testing instructions:

* First test the notice without this PR applied, on a Jetpack-connected site:
* Visit cloud.jetpack.com/pricing and select a plan, then complete the checkout.
* On the "Thank you for purchasing [product-name]!" page, just stop there.. (Don't enter a site URL in the input box).
* Visit the Jetpack dashboard - `/wp-admin/admin.php?page=jetpack#/dashboard`, and you should see a notice saying 'You have an available product license key. Activate it now, or view all your purchases.'
* Then test the above again with this PR applied - you should no longer see the notice on the Jetpack dashboard.
